pika/channel.py

```diff
@@ -1051,19 +1174,27 @@ def _on_getok(self, method_frame, header_frame, body):
         else:
             LOGGER.error('Basic.GetOk received with no active callback')
 
-    def _on_openok(self, frame_unused):
+    def _on_openok(self, method_frame):
         """Called by our callback handler when we receive a Channel.OpenOk and
         subsequently calls our _on_openok_callback which was passed into the
         Channel constructor. The reason we do this is because we want to make
         sure that the on_open_callback parameter passed into the Channel
         constructor is not the first callback we make.
 
-        :param pika.frame.Method frame_unused: Unused Channel.OpenOk frame
+        Suppress the state transition and callback if channel is already in
+        CLOSING state.
```

Comment:
REVIEWME change in `_on_openok` corresponding to this comment about suppressing.